WalletWasabi.Gui/Tabs/WalletManager/LoadWallets/LoadWalletViewModel.cs

```diff
@@ -280,21 +285,7 @@ private bool TrySetWalletStates()
 
 				IsWalletSelected = SelectedWallet != null;
 				CanTestPassword = IsWalletSelected;
-
-				if (Global.WalletManager.AnyWallet())
-				{
-					IsWalletOpened = true;
-					CanLoadWallet = false;
-				}
-				else
-				{
-					IsWalletOpened = false;
-
-					// If not busy loading.
-					// And wallet is selected.
-					// And no wallet is opened.
-					CanLoadWallet = !IsBusy && IsWalletSelected;
-				}
+				CanLoadWallet = SelectedWallet is { } ? (!SelectedWallet.IsBusy && SelectedWallet.WalletState <= WalletState.Initialized) : false;
```

Comment:
I think the parentheses are not necessary here.
```suggestion
				CanLoadWallet = SelectedWallet is { } ? !SelectedWallet.IsBusy && SelectedWallet.WalletState <= WalletState.Initialized : false;
```